File: JAVA/level1/2016년.java
class Solution {
  public String solution(int a, int b) {
      int[] day = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
      String[] dayWeek = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
      int date = 0;

      for(int i=0; i<a-1; i++)
          date += day[i];
      date += b;

      return dayWeek[date%7];
  }
}

File: JavaScript/level1/2016년.js
function solution(a, b) {
    var answer = '';
    let months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    let date = ['SUN', 'MON' , 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
    let days = b+4;
    
    for(let i=0; i<a-1; i++)
        days += months[i];
    
    return date[days%7];
}